src/reforms/tier_01/haafingar_kingdom.txt

```diff
@@ -209,4 +209,106 @@ governments:monarchy:reform_levels:1 += new tier_1_haafingar_kingdom = {
 	ai = {
 		factor = 1
 	}
+}
+static_modifiers:positive_stability += {
+	solitude_rock_hammer_influence = 0.1
+}
+new named_effect add_rock_hammer_building_influence = {
+	foreach $building in buildings {
+		$building:on_built += {
+			owner = {
+				add_faction_influence = {
+					faction = solitude_rock_hammer
+					influence = 1
+				}
+			}
+		}
+	}
+}
+new faction solitude_rock_hammer = {
+	name = "Rock-Hammer"
+	desc = "Faction Modifier:
+		{modifier_to_string = {
+			build_cost = -0.15
+			build_time = -0.25
+		}}
+§YGains influence through positive stability
+Gains influence through building§!
+"
+	monarch_power = ADM
+	modifier = {
+		build_cost = -0.15
+		build_time = -0.25
+	}
+}
+static_modifiers:naval_forcelimit += {
+	solitude_storm_galley_influence = 0.5
+}
+static_modifiers:mercantilism += {
+	solitude_storm_galley_influence = 1
+}
+new faction solitude_storm_galley = {
+	name = "Storm-Galley"
+	desc = "Faction Modifier:
+		{modifier_to_string = {
```

Comment:
modifier_to_string? what's changing from just modifier?

Reply:
modifier_to_string turns modifier code into player known names. So build_cost is changed to "Construction Time"